src/app_engine/app.yaml

```diff
@@ -9,6 +9,14 @@ handlers:
   static_files: html/\1
   upload: html/(.*\.html)
 
+- url: /(.*\.json)
+  static_files: html/\1
+  upload: html/(.*\.json)
```

Comment:
This makes me nervous because we are storing things like the gcm server key in json files. Could we add a specific file name here?